client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChangeEvent, useState } from 'react';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import DeleteIcon from '@material-ui/icons/Delete';
+import useStyles from './useStyles';
+import { User } from '../../interface/User';
+import ProfileAvatar from '../ProfileAvatar/ProfileAvatar';
+
+interface Props {
+  loggedInUser: User;
+}
+
+const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Paper elevation={5} className={classes.root}>
```

Comment:
Maybe also consider having a maxWidth for the container here so that the paper component does not stretch out?